docs/rfcs/0013-rfc-process-updates.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: RFC process updates
+permalink: /docs/rfcs/0013-rfc-process-updates
+---
+
+* RFC PR: https://github.com/mozilla-mobile/firefox-android/pull/5681
+* Start date: 2024-02-20 
+* End date: 2024-03-22 
+* Stakeholders: @jonalmeida, @boek
+
+## Summary
+
+Clarify requirements for RFCs by introducing a README and a template.
+
+## Motivation
+
+- The scale of the Firefox Android team has grown substantially since the RFC process was first introduced. 
+- More external teams are using and contributing to Android Components. 
+- Approval or rejection of a proposal has required periods of time that can slow implementation.
+- Stakeholders have either been implicit by sharing RFCs in various communication channels, or an author must be expected to correctly select reviewers that lack explicitly documented ownership of affected areas.
+
+## Guide-level explanation
+
+This proposal suggests improving the RFC process by introducing the following:
+
+- an RFC template
+- a guide for when RFCs are appropriate and when they are not needed in the form of a README
+- explicit stakeholders for RFCs
+- an initial deadline for each RFC
+- a new RFC section, "artifacts" which contain links to any additional artifacts introduced alongside the RFC
+
+## Artifacts
+
+[README](./README.md)
+[0000 RFC Template](./0000-template.md)
+[Follow-up bug for updating CODEOWNERS](https://bugzilla.mozilla.org/show_bug.cgi?id=1881373)
```

Comment:
```suggestion
* [README](./README.md)
* [0000 RFC Template](./0000-template.md)
* [Follow-up bug for updating CODEOWNERS](https://bugzilla.mozilla.org/show_bug.cgi?id=1881373)
```

nit: formatting.

Reply:
This list without references to what it is refering to can be confusing. For example, what does this README link reference here? Consider adding anchor links or annotations.

That said, I'm also not sure what the difference between this section is to the 'Prior Art' one.

Reply:
I tried to answer this a bit in [this comment](https://github.com/mozilla-mobile/firefox-android/pull/5681/files#r1498491586) as well, but I see "artifacts" as internal documentation or other resources that were generated as part of the proposal and "prior art" as reading into external sources that cover the same topic as the proposal